app/mattermost.js

```diff
@@ -44,6 +44,11 @@ const init = async () => {
 
     registerScreens(store, Provider);
 
+    if (__DEV__) {
+        const DevMenu = require('react-native-dev-menu');
+        DevMenu.addItem('StoryBook', () => goToScreen('StoryBook', 'StoryBook'));
+    }
```

Comment:
this is bothering me here for some reason... is there somewhere else where you can add it ?